tests/kafkatest/services/kafka/kafka.py

```diff
@@ -52,6 +51,7 @@ def advertised_listener(self, node):
     def listener_security_protocol(self):
         return "%s:%s" % (self.name, self.security_protocol)
 
+
```

Comment:
Is this intentional?

Reply:
This change not necessary.
But PyCharm and IDEA print warning without it: "PEP 8: E302 expected 2 blank lines, found 1"

I will revert it.